### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by boris-petrov**
cc @yuki24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Shadow DOM support for Capybara
 
-The `capybara-shadowm` gem adds basic support for the [Shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM) to Capybara.
+The `capybara-shadowdom` gem adds basic support for the [Shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM) to Capybara.
```
Comment (yuki24) [inline README.md:3]: 💥

File: README.md [hunk 2]
```diff
@@ -20,6 +20,12 @@ Or install it yourself as:
 
 ## Usage
 
+Require it in `spec_helper.rb`:
+
+```rb
+require 'capybara/shadowdom'
```
Comment (yuki24) [inline README.md:26]: Did you actually need this? I don't think this is required for most projects.

File: README.md [hunk 3]
```diff
@@ -41,6 +47,7 @@ within custom_element.shadow_root do
 
   # Asserts the text within the element:
   assert_text "Hello shadow world!"
+  expect(page).to have_text("Hello shadow world!")
```
Comment (yuki24) [inline README.md]: It's a little confusing to put both minitest and RSpec APIs. Could you add a new section for RSpec?

File: README.md (hunk 2 above)
Comment (boris-petrov) [reply README.md:26]: It seems so, yes. Otherwise it doesn't seem to override Capybara's default behavior.

File: README.md (hunk 3 above)
Comment (boris-petrov) [reply README.md]: Done.

**Comment by yuki24** [comment]
Thanks!

**Comment by boris-petrov** [comment]
Thank you for the plugin!

**Pull request merged**